ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -258,7 +258,9 @@ defDocumentSymbol l = DocumentSymbol { .. } where
   _detail         = Nothing
   _deprecated     = Nothing
   _name           = ""
-  _kind           = SkUnknown 0
+  -- This used to be SkUnknown 0, which is invalid, as SymbolKinds start at 1,
```

Comment:
Yeah, this is kind of gross, but it's clearly meant to be an arbitrary choice, so this seems okay.